source/includes/dev-mode-breaking-changes.rst

```diff
@@ -0,0 +1,11 @@
+*Applies to App Services Apps created after August 9th, 2023.* 
+
+Development Mode is not suitable for production use. If you use Development
+Mode, make sure to disable it before moving your app to production.
+
+App Services Apps in Development Mode that were created after August 9th, 2023
+can make :ref:`breaking changes <breaking-change-quick-reference>` from their
+client code to synced object schemas.
```

Comment:
I can't seem to find the grammar rules for why I think it should be this way, but I have in my head the "to" should come before the "from" in this sentence. This is definitely a nitpick, though, so YMMV.

```suggestion
can make :ref:`breaking changes <breaking-change-quick-reference>` to synced
object schemas from client code.
```

Reply:
🤣 I went back and forth on this sentence about a dozen times. I wanted to try to emphasize that the breaking changes are happening in the client, which is what the new thing is all about. If that seems too awkward, I'm happy to change it.